src/login/login.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+import { Context, useDefaultContext } from 'login/context'
+import { LoginMenu } from 'login/login-menu'
+
+export const Login = () => <Context.Provider value={useDefaultContext()}>
+  <LoginMenu />
+</Context.Provider>
```

Comment:
Would the name `index.jsx` be more relevant for the current file? I mean only the file, not the component!

Reply:
Done